config/featured.js

```diff
@@ -0,0 +1,295 @@
+export const listItems = [
```

Comment:
this file has all of the content. this definitely isn't the best way, but it's fine for now. it would be awesome to use a CMS of some sort that's _not_ wordpress 😂

Reply:
This is kind of a showstopper for me. This is about the most manual way we could go about this. I don't think this is a good idea. We're inserting ourselves as a middleman between the blog posts being published and the info getting to people.